src/utils/api.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+import Options from './options';
+
+const options = new Options();
+export default class API {
+  public async summaries() {
+    try {
+      const apikey = await options.getApiKeyAsync();
+      const token = Buffer.from(apikey).toString('base64');
+      axios.defaults.baseURL = 'https://wakatime.com';
```

Comment:
What about define it as constant `const baseURL = 'https://wakatime.com/api/'` ?

Reply:
i wanted to go as the wakatime docs shows for ecample look https://wakatime.com/developers#summaries
all the endPoints include the `api`, thats why i dont include it in the baseUrl
`GET` `/api/v1/users/:user/summaries`
`GET` `/api/v1/users/current/summaries`